alf/examples/muzero_atari_conf.py

```diff
@@ -427,30 +587,42 @@ def _scale_grad(scale):
     max_unroll_length=5,
     learn_policy_temperature=1.0)
 
+
+def model_ctor(*args, **kwargs):
+    model = SimpleMCTSModel(*args, **kwargs)
+    # Nero optimizer normalizes the parameters. We need to do this in the beginning
+    # so that the target_model is same as the model.
```

Comment:
Question: `target_model` is not optimized, why do we need to run `NeroPlus.initialize()` on it?

Reply:
> Question: `target_model` is not optimized, why do we need to run `NeroPlus.initialize()` on it?
Based on my understanding, `NeroPlus.initialize()` can be regarded as part of model's parameter initialization.

Reply:
I did not make it clear, sorry! The `model_ctor` is used to construct both the `model` and the `target_model`. It is necessary to run `NeroPlus.initialize()` when constructing the actual `model`. I think running this for `target_model` is not necessary but "it does not hurt". Not sure whether my understanding is correct.

Reply:
I think your understanding is correct: ``NeroPlus.initialize() ``  is not necessary for the target model but "it does not hurt".  In this sense, all the param initializations are actually unnecessary for the target model, since they will be wiped away by copying the values from model later.